webapp-django/crashstats/settings/bundles.py

```diff
@@ -194,10 +194,7 @@
         "output_filename": "js/select2.min.js",
     },
     "tablesorter": {
-        "source_filenames": (
-            "tablesorter/dist/js/jquery.tablesorter.js",
-            "crashstats/js/socorro/tablesorter_utils.js",
```

Comment:
I took out tablesorter-utils because we're going to switch to sorting all the hex strings as strings and left-padding the values.